tests/Services/Search/ParametersHandlerTest.php

```diff
@@ -19,6 +19,53 @@ public function getSortingOptionsProvider()
         return [
             'Sorting options for category page' => [
                 false,
+                false,
```

Comment:
When we are at it, please add keys to these data providers, so you can read the dataProvider without the need of the associated test. In this case:

```php
return [
    'Sorting options for category page' => [
        'isSearch' => false,
        'hasFilters' => false,
        // . . . 
    ]
]
```

Reply:
Please do this for all touched data providers.